Source/Blazorise/Components/NumericEdit/NumericEdit.razor.cs

```diff
@@ -92,7 +92,9 @@ protected override async Task OnFirstAfterRenderAsync()
                 Separator = DecimalsSeparator,
                 Step,
                 Min = Min.IsEqual( default ) ? minFromType : Min,
-                Max = Max.IsEqual( default ) ? maxFromType : Max
+                Max = Max.IsEqual( default ) ? maxFromType : Max,
```

Comment:
What if user provides a max that is higher then the type max? Do we care about that?

Reply:
Didn't I already fix this?

Reply:
Not sure. What I mean is, using numeric edit like this <NumericEdit TValue="int" Max="value higher then int max here" />

Reply:
There is no reason to support a value higher than int max because that same value cannot be stored in an `int`. You would store 300 into byte that has a max of 255. Same thing for the `int`.

Reply:
Yep I was just asking if we handle that use case correctly.

Reply:
Seems to me it is handled by the framework :)

![image](https://user-images.githubusercontent.com/900302/135820094-6ef05970-dfd5-4f59-83f5-2e1312258f02.png)

Since TValue is recognized it will give an error if we want to assign a value larger than it can handle. I also tried to trick it by casting but then it breaks at runtime. Which makes sense.

Reply:
Ah great, haha